src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -629,12 +676,15 @@ export class OnlineGameWrapperComponent extends GameWrapper implements OnInit, A
     public getPlayerNameFontColor(player: number): { [key: string]: string} {
         if (this.observerRole === player || this.observerRole > 1) {
             return { color: 'black' };
-        } else if (this.opponent && this.opponent.doc && this.opponent.doc.state === 'offline') {
+        } else if (this.opponentIsOffline()) {
             return this.OFFLINE_FONT_COLOR;
         } else {
             return { color: 'black' };
         }
     }
+    public opponentIsOffline(): boolean {
+        return this.opponent && this.opponent.doc && this.opponent.doc.state === 'offline';
```

Comment:
this.opponent != null && this.opponent.doc != null plutôt ?

Reply:
Oui, marrant qu'il ne se plaigne pas là de l'ambiguité, l'idée de notre rule lint c'était pas de se plaindre quand on utilise un non booléen comme valeur booléanisable ? (aka """prefer toBeStrictlyTrue to toBeTruthy""" ? )

Reply:
Par contre j'ai l'impression que thos.opponent.doc sera jamais null si this.opponent l'est pas, je garde quand même ?

Reply:
la rule a été désactivée pour être activée dans le CI (ticketé) il me semble. Sinon oui. si opponent est pas null, .doc ne l'est pas non plus donc à enlever

Reply:
C'était elle qui bizarrement mettait en fail toutes les premières lignes de chaque fichier ?

Reply:
oui!